### PR TITLE
update README ahead of UI one-click zero-config install

### PR DESCRIPTION
**Pull request by lindsaylevine**
Fixes https://github.com/netlify/netlify-plugin-nextjs/issues/44

this will definitely be a work in progress over the next day. please comment/suggest whatever.

### CODE REVIEW

**Review by rstavchansky — COMMENTED** [review]
@lindsaylevine I added a bunch of minor editing suggestions. Feel free to accept or not.
Looks good!

File: README.md [hunk 1]
```diff
@@ -11,49 +11,84 @@
   </a>
 </p>
 
-This build plugin is a utility for enabling server-side rendering in Next.js on Netlify. It wraps your application in a tiny compatibility layer, so that pages can use Netlify Functions to be server-side rendered.
+This build plugin is a utility for supporting Next.js on Netlify. To enable server-side rendering and other framework-specific features in your Next.js application on Netlify, your app will need to install this plugin.
```
[the remark this answers is not in the document]
Comment (ehmicky) [reply README.md:14]: I would also rename `app` to `site`?

File: README.md [hunk 2]
```diff
@@ -11,49 +11,84 @@
...
+  command   = "npm run build"
+
+[[plugins]]
+  package = "@netlify/plugin-nextjs"
+```
+
+This plugin can also be installed and managed from your site's settings on Netlify.
+If you want to install there, you can omit this [[plugins]] section from your .toml file.
+
+## Custom Netlify Functions
```
Comment (ehmicky) [inline README.md:41]: Do we even need to mention custom Netlify Functions at all?

I am assuming if a site is using a Functions directory, they will assume this plugin to just work with it.
If a site is not using a Functions directory, then the default Functions directory will be used instead, which the user will not see.

File: README.md [hunk 3]
```diff
@@ -11,49 +11,84 @@
...
-  publish = "out_publish"
+  functions = "my_functions_dir"
 
 [[plugins]]
   package = "@netlify/plugin-nextjs"
 ```
 
+[Read more about Netlify Functions here](https://docs.netlify.com/functions/overview/).
+
+## Publish Directory
```
Comment (ehmicky) [inline README.md:57]: Same comment as for Functions directory: do we even need to mention it?

**Review by ehmicky — COMMENTED** [review]
Thanks a lot @lindsaylevine!

File: README.md [hunk 4]
```diff
@@ -11,49 +11,84 @@
   </a>
 </p>
 
-This build plugin is a utility for enabling server-side rendering in Next.js on Netlify. It wraps your application in a tiny compatibility layer, so that pages can use Netlify Functions to be server-side rendered.
+This build plugin is a utility for supporting Next.js on Netlify. To enable server-side rendering and other framework-specific features in your Next.js application on Netlify, you will need to install this plugin for your app.
```
Comment (lindsaylevine) [inline README.md:14]:
```suggestion
This build plugin is a utility for supporting Next.js on Netlify. To enable server-side rendering and other framework-specific features in your Next.js application on Netlify, you will need to install this plugin for your site.
```

File: README.md [hunk 5]
```diff
@@ -11,49 +11,84 @@
...
+
+```toml
+[build]
+  command   = "npm run build"
+
+[[plugins]]
+  package = "@netlify/plugin-nextjs"
+```
+
+This plugin can also be installed and managed from your site's settings on Netlify.
```
[the remark this answers is not in the document]
Comment (verythorough) [reply README.md:39]: Yeah, I lean towards pitching the UI install first. We could start the installation section to say that the plugin can run with no configuration, and then present a link, probably in a text block or something to give it more attention, e.g.

> 👉 [**Click to install this plugin**](https://deploy-preview-6326--app.netlify.app/plugins/@netlify/plugin-nextjs/install) 👈 

I went a little goofy, but you get the idea. 😛 (Incidentally, I'm now going to file an issue for an "Install this plugin" button.)

For the link, after https://github.com/netlify/netlify-react-ui/pull/6326 is merged, this link will work:

http://app.netlify.com/plugins/@netlify/plugin-nextjs/install

Until then, you can see what the experience will be like by testing this link (which is what I used in my emoji-laden example:

https://deploy-preview-6326--app.netlify.app/plugins/@netlify/plugin-nextjs/install

**Review by verythorough — COMMENTED** [review]
Looking good! I'm posting my review now so you can see the comments so far. Might have another one or two.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (lindsaylevine) [reply README.md:39]: ok, i am going to handle the click to install link as a fast follow, but made a change suggesting ui install first pointing to the ui-based installation part of the docs !

File: README.md (hunk 2 above)
Comment (lindsaylevine) [reply README.md:42]: tabling for after <3

File: README.md [hunk 6]
```diff
@@ -11,49 +11,84 @@
...
 - [Showcase](#showcase)
 
 ## Installation and Configuration
 
-Create a `netlify.toml` in the root of your project:
+Create a `netlify.toml` in the root of your project. Your .toml should include:
+
+```toml
+[build]
+  command   = "npm run build"
```
[the remark this answers is not in the document]
Comment (lindsaylevine) [reply README.md:32]: tabling for after <3

**Review by verythorough — APPROVED** [review]

**Pull request merged**